cli/node/daemon.go

```diff
@@ -129,6 +129,10 @@ func (d *socketDaemon) handleConn(conn net.Conn) {
 		return
 	}
 
+	dela.Logger.Debug().
```

Comment:
question: is the debug level off by default?

Reply:
That's going to be documented but there is an environment variable to enabled them `LLVL=trace go test` so that they can be turned on on-demand. Otherwise it can be confusing when having warnings in the tests (but it's expected).